source/gloperate/source/pipeline/Stage.cpp

```diff
@@ -40,7 +42,12 @@ Stage::Stage(Environment * environment, const std::string & className, const std
 : cppexpose::Object((name.empty()) ? className : name)
 , m_environment(environment)
 , m_alwaysProcess(false)
+, m_measurementCycle(0)
+, m_measurementHistorySize(10) //must be even
```

Comment:
Add explanation of restriction. This information is suitable for the member documentation, too.